test/lit.cfg

```diff
@@ -833,3 +853,6 @@ config.substitutions.append(('%target-object-format', config.target_object_forma
 
 config.substitutions.append(('%target-resilience-test-wmo', config.target_resilience_test_wmo))
 config.substitutions.append(('%target-resilience-test', config.target_resilience_test))
+
+config.substitutions.append(('%target-llvm-profdata', os.path.join(llvm_tools_dir, 'llvm-profdata')))
+config.substitutions.append(('%target-llvm-cov', os.path.join(llvm_tools_dir, 'llvm-cov')))
```

Comment:
I've dropped the 'target-' prefix and added:

    config.llvm_profdata = inferSwiftBinary('llvm-profdata')
    config.llvm_cov = inferSwiftBinary('llvm-cov')